pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -604,6 +759,26 @@ func (lbaas *LbaasV2) UpdateLoadBalancer(clusterName string, service *api.Servic
 			if err != nil && !isNotFound(err) {
 				return err
 			}
+			// Delete the rules in the Node Security Group
```

Comment:
(Changed my mind and I'm restoring a deleted comment)
Why is this here?  There's nothing node-specific about the rules we're about to delete.